packages/plugin-table/src/TableRowXmlDomParser.ts

```diff
@@ -45,15 +51,21 @@ export class TableRowXmlDomParser extends AbstractParser<Node> {
 
         // Parse the <tbody> or <thead>'s attributes into a technical key of the
         // node's attributes, that will be read only by `TableRowDomRenderer`.
-        const containerAttributes = this.engine.parseAttributes(tableSection);
+        const attributes = this.engine.parseAttributes(tableSection);
+        let containerAttributes;
+        if (attributes) {
+            containerAttributes = new TableSectionAttributes(attributes);
+        }
```

Comment:
I don't like this construction. I didn't like it for ListItem and I still don't like it here.
Why not:
```typescript
const attributes = this.engine.parseAttributes(tableSection);
```
Then later in the code, when you *actually* need it:
```typescript
new TableSectionAttributes(attributes);
```